package/CHANGELOG

```diff
@@ -13,6 +13,25 @@ The rules for this file:
   * release numbers follow "Semantic Versioning" http://semver.org
 
 ------------------------------------------------------------------------------
+03/21/22 aya9aladdin
```

Comment:
Unfortunately this isn't how we add things to the changelog either, entries should be under 2.2.0 (see below). You should add your username under the same line that starts with mine, and then the changelog entry under the relevant section after that.